### ISSUE TITLE
WSL - Unable to open folder from UI

### ISSUE THREAD

**Issue by andreamah** [microsoft/vscode-remote-release#9709]
1. Connect to WSL
2. Try to open folder from `Open Folder` button.
3. Click through to your folder
4. Notice that you can't open the folder until you manually remove the trailing slash.

https://github.com/microsoft/vscode-remote-release/assets/31675041/f08b8c0b-9b37-4e94-ac0b-33d9877f1944

Version: 1.88.0-insider (user setup)
Commit: 246d700c4604eb5ebdbb561a1a86562bf9217a62
Date: 2024-03-27T05:50:10.528Z
Electron: 28.2.6
ElectronBuildId: 27476517
Chromium: 120.0.6099.291
Node.js: 18.18.2
V8: 12.0.267.19-electron.0
OS: Windows_NT x64 10.0.22631

**Comment by aeschli** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/208937
